src/OSPSuite.Core/Import/DataFormatParameter.cs

```diff
@@ -171,7 +171,7 @@ public override bool IsGroupingCriterion()
 
       public override bool ComesFromColumn()
       {
-         return false;
+         return true;
```

Comment:
Lol. Drastic change. :)

Reply:
yeap, was wrong before, although it was not causing any errors